src/Justimmo/Model/Query/AbstractQuery.php

```diff
@@ -149,6 +149,9 @@ public function findPk($pk)
         if (isset($this->params['picturesize'])) {
             $params['picturesize'] = $this->params['picturesize'];
         }
+        if (isset($this->params['alleProjektObjekte'])) {
+            $params['alleProjektObjekte'] = $this->params['alleProjektObjekte'];
+        }
```

Comment:
der parameter funktioniert nur bei projekten, macht es probleme/fehler wenn es bei den anderen queries verwendet wird?

Reply:
`ProjectQuery::allProjectRealties()` und `RealtyQuery::allProjectRealties()` Methoden können diesen Parameter setzen und es ist für beide gültig.
Wenn das Checkbox "_Untereinheiten von Wohnbauprojekten unabhängig vom Objektstatus exportieren_" nicht aktiviert ist, wird der Parameter in der Abfrage ignoriert.

Für normale Objekte sehe ich keine Unterschiede zwischen den Abfragen und APIv1 erlaubt beides:
https://api.just.immo/rest/v1/objekt/detail?objekt_id=11218665&culture=de
https://api.just.immo/rest/v1/objekt/detail?objekt_id=11218665&culture=de&alleProjektObjekte=1

Für Projekte gibt es eine zweite Abfrage, die in den beiden Fällen etwas anders ist, aber ich konnte beide aufrufen. Das 2. enthielt offensichtlich mehr Unterobjekte:
https://api.just.immo/rest/v1/projekt/detail?id=11111373&culture=de-at
https://api.just.immo/rest/v1/projekt/detail?id=11111373&culture=de-at&alleProjektObjekte=1